verification/curator-service/api/src/model/user.ts

```diff
@@ -14,6 +14,10 @@ const userSchema = new mongoose.Schema({
         enum: userRoles,
     },
     picture: String,
+    hasSeenWelcomePopup: {
+        type: Boolean,
+        default: false,
```

Comment:
isn't the default false for a boolean?

Reply:
I don't see any mention in the docs of what the default value for booleans is. It could potentially be `undefined` as well. However clients should be prepared to deal with an undefined value, so I removed the default here.